packages/config/src/utils.ts

```diff
@@ -48,3 +48,7 @@ export function formatId(id: number | string): string {
 export function padVersion(version: string): string {
 	return version + ".0";
 }
+
+export function stringify(data: any, beautify: boolean = true): string {
+	return JSON.stringify(data, undefined, beautify ? 4 : 0);
+}
```

Comment:
That method already exists here: `packages\shared\src\strings.ts`
You can add the `beautify` param there if you need it.